ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -613,6 +613,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: merge-gke-version-compatibility-tests
+  postsubmit: true
+  steps:
+    test:
+    - ref: stackrox-e2e
 - as: merge-gke-postgres-qa-e2e-tests
   postsubmit: true
   steps:
```

Comment:
(not about this line)
The PR title claims that it enables this test in nightlies, but I see `gke-version-compatibility-tests` mentioned in `ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml` file. Is PR title inaccurate or something else needs to be done to fully enable them on nightlies?

Reply:
The PR title is inaccurate in this case because it was already enabled for nightlies. I usually just take the JIRA issue title and copy it over as the PR title but I can remove the nightly part from it.